modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -51,6 +51,11 @@ The following snippet ensures that the interface has no IP address:
 # ...
 ----
 
+[IMPORTANT]
+====
+If you set the `ipv4.enabled` or the `ipv6.enabled` parameter to `false`, do not set the `state` parameter to `down`. The `state: down` configuration sets the `autoconnect` parameter for the Network Manager to `false`. This means the interface that you want to disable still receives a DHCP IP address. 
```

Comment:
I think I would kind of reverse the order here. What we're trying to communicate is never to use state: down, but instead to use state: up with ipv4 and ipv6 enabled: false. The part about autoconnect might be more detail than most people need too. I barely understand what's going on there. :-)

Reply:
Thanks, @cybertron .

Understandable. I updated the clause. Does it read better now?